packages/api-plugin-orders/src/schemas/schema.graphql

```diff
@@ -112,6 +112,12 @@ extend type Query {
     "A valid anonymous access token for this order. Required if the order is not linked with an account."
     token: String
   ): [Refund]
+
+  "Validate the order for correctness & completeness before placing the order"
+  validateOrder(
+    "input with order and payment objects similar to the one used for placeOrder"
+    input: validateOrderInput!
+  ): validateOrderPayload!
```

Comment:
`ValidateOrderInput` and `ValidateOrderPayload` should begin with uppercase letters by convention.